cpp/tests/utilities/cudf_test_fixtures.h

```diff
@@ -48,3 +50,34 @@ struct GdfTest : public ::testing::Test
         ASSERT_RMM_SUCCEEDED( rmmFinalize() );
     }
 };
+
+struct TempDirTestEnvironment : public ::testing::Environment
+{
+    char* tmpdir;
+    public:
+
+    void SetUp() {
+        tmpdir=new char[30];
+        strcpy(tmpdir, "/tmp/gtest.XXXXXX");
+        tmpdir=mkdtemp(tmpdir);
+        strcat(tmpdir, "/");
+    }
+
+    void TearDown() {
+        nftw(tmpdir, rmFiles ,10, FTW_DEPTH|FTW_MOUNT|FTW_PHYS);
```

Comment:
Once we can support c++17 should use `std::filesystem` instead.

Reply:
added as TODO comment.